src/hotspot/share/classfile/classLoader.cpp

```diff
@@ -509,19 +509,19 @@ void ClassLoader::setup_bootstrap_search_path(TRAPS) {
   } else {
     trace_class_path("bootstrap loader class path=", sys_class_path);
   }
-  setup_bootstrap_search_path_impl(sys_class_path, CHECK);
+  setup_bootstrap_search_path_impl(current, sys_class_path);
 }
 
 #if INCLUDE_CDS
-void ClassLoader::setup_app_search_path(const char *class_path, TRAPS) {
+void ClassLoader::setup_app_search_path(Thread* current, const char *class_path) {
   Arguments::assert_is_dumping_archive();
 
   ResourceMark rm;
```

Comment:
You can add 'current' as a parameter to ResourceMark since you have it.

Reply:
Thanks Coleen.
I've fixed it.